src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -141,7 +152,7 @@ export default function (props: ConnectDevicesMenuProps) {
 
     console.log("settings", deviceExternalAccounts);
```

Comment:
Pre-existing condition, but I see a few debug console logs got left in this file at some point.